CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 # Unreleased
 
+## [7.0.0] - Tue Jul 17 2018
+### Change
+- Remove Windows features support since them are not so long supported in RHMAP studio.
```

Comment:
Suggestion for better wording:
"Removed support for Windows platform since it's no longer supported in RHMAP studio."
P.S: I am not a native speaker, so take this with a grain of salt.

Reply:
The problem with "Removed support for Windows platform" is that the users will not able to use fh-fhc in a win SO. Because of it "Remove Windows features support" instead of "Removed support for Windows platform"

Reply:
Ok. Anyway, your sentence doesn't seem correct to me.
Remove Windows features support since **them** are **not so long** supported in RHMAP studio.
IMHO should be:
Remove Windows features support since **they** are **no longer** supported in RHMAP studio.

Reply:
You are correct. Please, could approve this one with the text fixed https://github.com/feedhenry/fh-fhc/pull/450